Example/test_compute_distance.py

```diff
@@ -1,14 +1,14 @@
 from compute_distance import distance_to_destination
 
 def test_distance_to_destination_4():
-    assert(distance_to_destination(22, 26) == 14)
+    assert(distance_to_destination(22, 26) == 4)
```

Comment:
great

Reply:
thank you sir.. what about the Exercise any comments